Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -200,30 +211,60 @@ private void Update()
                     {
                         rightPoint.transform.position = GetContactForHand(Handedness.Right);
                     }
+
+                    
                 }
             }
         }
         #endregion MonoBehaviour Handlers
 
 
         #region Private Methods
+        private bool TryGetMRControllerRayPoint(HandPanData data, out Vector3 rayPoint)
+        {
```

Comment:
Any reason you are doing a string comparison here? We've been trying to avoid determining controller types by doing string comparison, since the name could easily change.

If you really want to check if it is a mixed reality controller, why not just do `if (currentController is WindowsMixedRealityController)`?

But, more generally, is this restricting you to only work with MR controllers? Would slate then break if you used an oculus, or a vive?

Reply:
is there a more general generic controller type than WindowsMixedRealityController?

Reply:
got rid of string compares and Pointers[0] convention